README.md

```diff
@@ -30,15 +30,14 @@ Feel free to [open issues](https://github.com/mhammond/pywin32/issues) for
 all bugs (or suspected bugs) in pywin32. [pull-requests](https://github.com/mhammond/pywin32/pulls)
 for all bugs or features are also welcome.
 
+However, please **do not open GitHub issues for general support requests**, or
+for problems or questions using the modules in this package. For such requests, please start a [discussion](https://github.com/mhammond/pywin32/discussions) under the [Q&A category](https://github.com/mhammond/pywin32/discussions/new?category=q-a). This is now preferred to the
+[python-win32 mailing list](https://mail.python.org/mailman/listinfo/python-win32).
```

Comment:
I don't want to steer people away from the mailing list - maybe reword this to point more general "windows help" that way and specific pywin32 stuff here?